README.md

```diff
@@ -1,10 +1,45 @@
 # Theia
 
-Theia contains Network Flow Visibility functions which are exacted from the
-[Antrea main repo](https://github.com/antrea-io/antrea). While flow exporter and
-flow aggregator are kept in the Antrea repo, other flow visibility modules, such
-as ClickHouse and Grafana related ones, will be moved to this new repo.
-
-During the code migration period, we will still keep related functions available
-and stable in the Antrea repo, and the functions in this new repo won't be ready
-until we announce the migration is completed.
+Theia is a network observability and analytics platform for Kubernetes. It is
+built on top of [Antrea](https://github.com/antrea-io/antrea), and consumes
+[network flows exported by Antrea](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to provide fine-grained visibility into the network communications in a
+Kubernetes cluster.
+
+## Getting Started
+
+Getting started with Theia is simple. You can follow the [Getting Started](docs/network-flow-visibility.md#getting-started)
+guide to install Theia and start rocking!
+
+Theia supports network flow visualization and monitoring with Grafana. Check the
+[Grafana Flow Collector](docs/network-flow-visibility.md#grafana-flow-collector)
+document to learn more.
+
+Based on the collected network flow information, Theia can recommend appropriate
+NetworkPolicy configuration to secure Kubernetes network and applications.
+Please refer to the [NetworkPolicy Recommendation](docs/networkpolicy-recommendation.md)
+user guide to learn more.
+
+## Contributing
+
+The Antrea community welcomes new contributors. We are waiting for your PRs!
+
+* Before contributing, please get familiar with our
+[Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
Took a look at the [workflow failure](https://github.com/antrea-io/theia/runs/6866334327?check_suite_focus=true), it seems the `CODE_OF_CONDUCT.md` is not added? Or shall we pointing to the file in Antrea repo?

Reply:
It will be added in #40